src/network/networkd-lldp-tx.c

```diff
@@ -240,8 +240,6 @@ static int link_send_lldp(Link *link) {
         (void) parse_env_file("/etc/machine-info", NEWLINE, "PRETTY_HOSTNAME", &pretty_hostname, NULL);
 
         ttl = DIV_ROUND_UP(LLDP_TX_INTERVAL_USEC * LLDP_TX_HOLD + 1, USEC_PER_SEC);
-        if (ttl > (usec_t) UINT16_MAX)
-                ttl = (usec_t) UINT16_MAX;
```

Comment:
I think the idea is that `LLDP_TX_HOLD` or one of the other variables might be changed in the future resulting in a silent error here. The `if` is indeed not very useful, but I think it should be changed to `assert_cc`.